src/main/java/org/itsallcode/openfasttrace/Reporter.java

```diff
@@ -125,4 +122,24 @@
      *            reporter plug-in)
      */
     void reportToStdOutInFormat(final Trace trace, final String format);
+
+    /**
+     * Get the default report format
+     * 
+     * @return <code>String</code> representing the default report format.
+     */
+    public static String getDefaultReportFormat()
+    {
+        return "plain";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this method and declare a constant for this value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3400)